torch_tvm/fusion_pass.cpp

```diff
@@ -59,9 +63,22 @@ c10::optional<Node*> tryMerge(
       consumer->kind().toQualString(),
       ":\n");
 
-  // Symbolic checks
-  REQ(canHandle(producer, aliasDb));
-  REQ((canHandle(consumer, aliasDb) || consumer->kind() == getTVMSymbol()));
+  // if producer cannot be converted, lower consumer to TVM
+  if(!canHandle(producer, aliasDb)){
+    if(consumer->kind() == getTVMSymbol() || consumer->hasAttribute(attr::Subgraph)) {
+      // Already converted so return no change
+      return c10::nullopt;
+    }
+    // proceed to convert current node to TVM
+    if(!aliasDb.isMutable(consumer)){
+      REQ(!aliasDb.hasOutputWriters(consumer));
+    }
+    consumer = SubgraphUtils::createSingletonSubgraph(consumer, getTVMSymbol());
```

Comment:
I also meant to suggest that we not only check if it can be lowered, but if it can be we create the singleton graph outside. Thus tryMerge always has consumer that is already lowered and it only does merging but never creation.
I think it will be cleaner this way. Sorry if it feels like I am dragging much want to drag it much.